### ISSUE TITLE
Alt Text image product not working on 2.0.2

### ISSUE THREAD

**Issue by binamandiri** [magento/magento2#3322]
Alt Text image product not working on 2.0.2  ??
if this cant work it's not good for SEO images product.

**Comment by katmoon** [comment]
Hi,

Thank you for reporting the issue!
Internal ticket number is `MAGETWO-31691`

Thanks,
Kateryna.

**Comment by katmoon** [comment]
The issue is fixed in the latest Magento version.
I'm closing the ticket. Please feel free to reopen if you have additional questions.

**Comment by johnny-longneck** [comment]
This is still existing.

**Comment by timste** [comment]
in category listing M2 use the name from the product for the alt tag. in product detail view, when you dont set the alt text is the alt tag missing. in M1 was the default alt text the product name.
